__tests__/server.test.js

```diff
@@ -111,3 +111,36 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("should return code 200 on successful call", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("should return all objects in articles with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).not.toBe(0);
+        body.articles.forEach((article) => {
+          expect(article).toHaveProperty("author");
```

Comment:
We can make this more thorough by checking the type of the property values. Please add this in

Reply:
Would that just be a case of adding expect(typeof article.property).toBe or is there a different method?